app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -330,6 +330,13 @@ input.top-menu-search--input
     @include varprop(font-size, header-item-font-size)
     text-decoration: none
     padding: 0 15px
+  &:not(.-collapsed)
+    @include varprop(background, header-item-bg-hover-color)
+  &:hover
+    @include varprop(background, header-item-bg-hover-color)
+    &:not(.-collapsed)
+      i::before
+        content: "\f12c"
```

Comment:
The icon code will change when we update the icon font the next time. Please use `@include icon-mixin-close`